RFC-004-SWAP-Bitcoin.md

```diff
@@ -0,0 +1,78 @@
+# Bitcoin Ledger in SWAP Messages
+
+- RFC-Number: 004
+- Status: Draft
+- Discussion-issue: -
+- Created on: 1 Mar. 2019
+
+# Table of contents
+
+<!-- toc -->
+
+- [Description](#description)
+- [Motivation](#motivation)
+- [Content](#content)
+  - [Ledger definition](#ledger-definition)
+    - [Parameters](#parameters)
+  - [Asset definition](#asset-definition)
+- [Registry extension](#registry-extension)
+
+<!-- tocstop -->
+
+## Description
+
+This RFC introduces Ledger & Asset definitions to be used in SWAP Messages for [Bitcoin Core](https://github.com/bitcoin/bitcoin/) network.
+
+## Motivation
+
+This RFC aims to define the necessary values and parameters to support Bitcoin Core in SWAP messages.
+This, added to complimentary RFCs per SWAP protocol, allows the execution of SWAP involving assets defined on the Bitcoin Core Ledger.
+
+## Content
+
+### Ledger definition
+
+This RFC's objective is to uniquely identify the Ledger on which the SWAP will operate.
+It is a difficult task as soft and hard forks may happen and there is no guarantee that the conditions identifying today are valid tomorrow.
+
+We define Bitcoin Core as the network and consensus whose standard implementation is [bitcoind](https://github.com/bitcoin/bitcoin/).
+
+- Name: Bitcoin
+- Description: Bitcoin Core Network
+- `value`: `bitcoin`
```

Comment:
IMO, it is a bit too implicit to just list `value` here. Which `value` is meant?

Maybe mention where this comes from and how it is related:

```
RFC002 defined the headers `alpha_ledger` and `beta_ledger` which follow the standard header format defined in RFC001.
```

Reply:
The headers already refer back to Ledger definition. I don't feel the need to re-refer and re-define.

Reply:
It is not about re-defining. It is more about establishing a context for the reader so that it reads nicely and there is not a bullet points list appeary in the middle of the text flow.

Reply:
For example, I really liked the way @LLFourn wrote this RFC: https://github.com/comit-network/RFCs/blob/master/RFC-003-SWAP-basic.md

Reply:
note: I am going for the explicit verbal approach suggested by @thomaseizinger .